### PR TITLE
Created my card

### PR DESCRIPTION
**Pull request by TheJSPythonator**
I created my card.

### CODE REVIEW

**Review by CandidDeer — CHANGES_REQUESTED** [review]
Please add a social media handle. This can bei any Platform, even your GitHub. Thx.

**Comment by TheJSPythonator** [comment]
Ok!

File: index.html [hunk 1]
```diff
@@ -105,7 +105,31 @@ <h2>A project for first-time contributions</h2>
...
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-        
+
+        <!-- ________ TheJSPythonator Contributor card START ________  -->
+        <div class="card">
+          <p class="name">TheJSPythonator</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/TheJSPythonator" target="_blank">Github</a>-->
```
Comment (TenTraicion) [inline index.html:114]:
```suggestion
            <a href="https://github.com/TheJSPythonator" target="_blank">@TheJSPythonator</a>
```

File: index.html [hunk 2]
```diff
@@ -105,7 +105,31 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">Hello! I code in JS and Python. I love reading, writing, drawing and playing Tekken (usually TTT2). I taught myself coding using online resources.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://codetheweb.blog/" target="_blank" title="Code The Web">Code The Web</a>
+              </li>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" title="MDN Web Docs">MDN Web Docs</a>/a>/a>/a>/a>a>
```
Comment (TenTraicion) [inline index.html:124]:
```suggestion
                <a href="https://developer.mozilla.org/en-US/" title="MDN Web Docs">MDN Web Docs</a>
```

File: index.html [hunk 3]
```diff
@@ -105,7 +105,31 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="https://www.sitepoint.com/" target="_blank" title="SitePoint">SitePoint</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________ TheJSPythonator Contributor card END ________  -->
          <!-- ========= Start ========= -->
```
Comment (TenTraicion) [inline index.html:133]: move your card after this line where it says, Start
make sure your card has a line break after and before.

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@TheJSPythonator,
Kindly make the following changes, so that we can merge your card. Thanks..!

**Pull request closed without merging**